database/migrations/2020_05_11_100000_create_participants_table.php

```diff
@@ -12,11 +12,12 @@ public function up()
     {
         Schema::create('participants', function (Blueprint $table) {
             $table->id();
-            $table->foreignIdFor(app('user'))->nullable()->index();
+            $table->morphs('user')->nullable();
             $table->string('email')->unique();
-            $table->string('name')->nullable(); // first name
-            $table->string('name_last')->nullable();
-            $table->date('dob')->nullable(); // date of birth
+            $table->string('first_name')->nullable();
+            $table->string('last_name')->nullable();
+            $table->date('birth_date')->nullable();
+            $table->tinyInteger('is_verified')->nullable();
             $table->softDeletes(); // Soft delete if the participant email address bounces
             $table->timestamps();
         });
```

Comment:
The schema changes will create breaking updates to the production database. I need to keep an eye on this to migrate the existing data correctly.